Sources/DeepLearning/Optimizer.swift

```diff
@@ -91,11 +112,24 @@ public class Adam<Model: Layer, Scalar: TensorFlowFloatingPoint>: Optimizer
     }
 }
 
+/// RMSProp optimizer.
```

Comment:
```suggestion
/// The RMSProp optimizer.
```

Reply:
I deliberately chose not to add an article, actually.
I think `The xxx optimizer` is not quite appropriate because there can exist multiple distinct instances of `RMSProp`.

I would prefer `RMSProp optimizer` over `An optimizer that implements the RMSProp algorithm` over `An RMSProp optimizer` over `The RMSProp optimizer`.

Reply:
Okay.